tests/Unit/Transitions/Update.test..ts

```diff
@@ -0,0 +1,57 @@
+import JsonResponse from "../../../src/Interfaces/JsonResponse";
```

Comment:
This test doesn't run... You can confirm with a coverage report or by checking the output in either your terminal or Actions